src/routes/business/legalRepresentative.ts

```diff
@@ -0,0 +1,70 @@
+import type { Request, Response } from "express";
+import moment from "moment";
+
+import generateID from "../../helpers/id";
+import { getBusiness, saveBusiness } from "../../db";
+import uuid from "node-uuid";
+import {
+  LegalRepresentative,
+  LegalRepresentativeType,
+  LegalRepresentativeRepresentationType,
+} from "../../helpers/types";
+
+export const createLegalRepresentative = async (
+  req: Request,
+  res: Response
+) => {
+  const { business_id: businessId } = req.params;
```

Comment:
by using `withBusiness` middleware, you can get business entity directly from request